code/modules/clothing/gloves/stungloves.dm

```diff
@@ -88,9 +92,10 @@
 	else
 		L.visible_message(SPAN_DANGER("[L] has been punched with [src] by [user]!"))
 	playsound(loc, 'sound/weapons/Egloves.ogg', 50, 1, -1)
-	L.stun_effect_act(stun, agony, user.targeted_organ, src)
-	msg_admin_attack("[key_name(user)] stunned [key_name(L)] with the [src].")
-	deductcharge(hitcost)
+
+	if(deductcharge(hitcost))
+		L.stun_effect_act(stun, agony, user.targeted_organ, src)
+		msg_admin_attack("[key_name(user)] stunned [key_name(L)] with the [src].")
 
```

Comment:
Not sure admins should be messaged upon a stun, attack logs should be enough.

Reply:
I didn't add this, it was already there before hand, did you want me to remove it still?

That being said, I don't know if attack logs register for help intent and the admins should know whether the stun actually went through probably

Reply:
Just checked apparently it doesn't message admins or make a log entry, either way im looking into it now

Reply:
nevermind the msg_admin_attack thing is a preference thing, and this IS needed for that, an important issue on the side though is still that stun gloves arent logged in the attack logs

Reply:
fixed